examples/mongoose.test.js

```diff
@@ -2045,4 +2045,54 @@ describe('Mongoose', function() {
     assert.equal(profilingLevel, 'off');
     // acquit:ignore:end
   });
+  describe('mongoose-find-async', function() {
+    it('find', async function() {
+        // acquit:ignore:start
+        await mongoose.disconnect();
```

Comment:
No need to keep doing `mongoose.disconnect()` and `mongoose.connect()` in examples. We can omit that because it is fairly standard for Mongoose.